### DISCUSSION TITLE
Importer: additionally add all device configs to local git repo

### DISCUSSION THREAD

**Discussion by tpurschke** [CactuseSecurity/firewall-orchestrator#2101] [General]
- add local git repo on backend server
- do this in addition to DB Import
- do not write the full (raw) config to db table import_full_config but to git
- also add Import id to git
- might also be used to add routing capabilities later
- git diff saving function should make sure not to collect too much data
- do not save any unchanged configs
- do not save binaries
- to be used as a minimal starting point for all (future) import modules

**Comment by NilsPur** [comment]
@tpurschke phase2 ?
